docs/architecture/cloud-native/distributed-data.md

```diff
@@ -1,39 +1,42 @@
 ---
-title: Distributed data
-description: Architecting Cloud Native .NET Apps for Azure | Distributed Data for Cloud Native Apps
-ms.date: 06/30/2019
+title: Cloud-native data
+description: Contrast data storage in monolithic and cloud-native applications
+author: robvet
+ms.date: 10/21/2019
 ---
-# Distributed data for cloud-native apps
+# Cloud-native data
 
-[!INCLUDE [book-preview](../../../includes/book-preview.md)]
```

Comment:
Why remove preview? We can remove this once we have completed the review for the entire book.